### PR TITLE
pfetch-rs 2.2.0 (new formula)

### PR DESCRIPTION
**Pull request by 21st-centuryman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by 21st-centuryman** [comment]
So I understand the issue with the project being too young but the source url being unreachable is tricky to me. 

Do i need to change https://github.com/Gobidev/pfetch-rs to https://github.com/gobidev/pfetch-rs?

I will rewrite the bottle do and re-submit this when the project is a bit older.

File: Formula/pfetch-rs.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class PfetchRs < Formula
+  desc "Pretty system information tool written in Rust"
+  homepage "https://github.com/Gobidev/pfetch-rs"
+  url "https://github.com/Gobidev/pfetch-rs/archive/refs/tags/v2.2.0.tar.gz"
+  sha256 "b21a91a9536ca776c38541e3d79a6c8fab4b522db0ef15353cb11e0833e7d3f7"
+  license "MIT"
+  head "https://github.com/Gobidev/pfetch-rs", branch: "main"
```
Comment (ZhongRuoyu) [inline Formula/pfetch-rs.rb:7]: You need a `.git` here:

```suggestion
  head "https://github.com/Gobidev/pfetch-rs.git", branch: "main"
```

Comment (21st-centuryman) [reply Formula/pfetch-rs.rb:7]: thank you for the quick reply. I will close this PR and wait for the project to mature. 

Have a great day.

File: Formula/pfetch-rs.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class PfetchRs < Formula
+  desc "Pretty system information tool written in Rust"
+  homepage "https://github.com/gobidev/pfetch-rs"
+  url "https://github.com/gobidev/pfetch-rs/archive/refs/tags/v2.2.0.tar.gz"
```
Comment (ZhongRuoyu) [inline Formula/pfetch-rs.rb:4]: You don't need to change these, actually. The audit was complaining about the `HEAD`.

```
HEAD: The source URL https://github.com/Gobidev/pfetch-rs is not reachable
```

```suggestion
  homepage "https://github.com/Gobidev/pfetch-rs"
  url "https://github.com/Gobidev/pfetch-rs/archive/refs/tags/v2.2.0.tar.gz"
```

Comment (21st-centuryman) [reply Formula/pfetch-rs.rb:4]: Yes I saw your comment after tge fact.

**Comment by ZhongRuoyu** [comment]
> I will rewrite the bottle do and re-submit this when the project is a bit older.

Thanks @21st-centuryman. You may open a new PR when ready.

**Pull request closed without merging**